Big-Spender/readme.md

```diff
@@ -85,6 +88,7 @@ INSERT YOUR QUERY HERE
 
 ```sql
 CREATE YOUR QUERY HERE
+SELECT SUM(amount),date FROM spends GROUP BY date;
```

Comment:
We are grouping by date, but what if the date has other days throughout the month? ie- 2021-04-15 -- would this still work?